webapp/components/survey/NodeDefDetails/BasicProps/store/useIsDisplayInParentPageDisabled.js

```diff
@@ -0,0 +1,8 @@
+import * as NodeDefLayout from '@core/survey/nodeDefLayout'
+
+import { useSurveyCycleKey } from '@webapp/store/survey'
+
```

Comment:
do really need this hook? u just set a property in the useBasiProps?

Reply:
u r right